mitodl/mitxpro
# Added recaptcha to register page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #151 

NOTE: did not handle any styling of this form since that's being done in a separate issue (#247)

#### What's this PR do?
Adds the recaptcha UI to the register page

#### How should this be manually tested?
- Configure `RECAPTCHA_SITE_KEY` and `RECAPTCHA_SECRET_KEY` with the corresponding settings from a recpatcha configured for your local hostname of choice at https://www.google.com/recaptcha
- Restart your web container
- Go to /signup and verify you see the recaptcha and that you can't proceed without completing it

#### Screenshots (if appropriate)

![Screenshot_2019-05-17 mitxpro(1)](https://user-images.githubusercontent.com/28598/57941985-594e9980-789e-11e9-98e7-62aa6e50aef8.png)

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/fyc0IZqqFxspW/giphy.gif)
